<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bosspage</class>
 <widget class="QWidget" name="bosspage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="timeOrder">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>345</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>消费时段分布</string>
   </property>
  </widget>
  <widget class="QLabel" name="bar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>老板系统</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bar_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询系统信息</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="viewAllInfo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看系统信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewAllOrder">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>345</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看流水明细</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dishRanking">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>菜品排行榜</string>
   </property>
  </widget>
  <widget class="QLabel" name="bar_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据统计分析</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
